packages/react-vanilla-components/src/components/tabs/HorizontalTab.tsx

```diff
@@ -39,36 +37,59 @@ const HorizontalTab = (props: PROPS_PANEL) => {
     }, { v: [] });
 
   const handleClick = (index: number) => {
-    setActiveTabIndex(index);
+    setActiveTab(index);
   }
 
-  const handleButtonClick = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
-    event.preventDefault();
-    setShortDescription(!shortDescription);
-    setLongtDescription(!longDescription);
-  };
-
   const getChild = useCallback((child: any, index: number) => {
     const Comp = getRenderer(child, mappings);
     return Comp ? <Comp key={`${child.id}_${index}`} {...child} /> : (null);
   }, []);
 
   return (
-    <div data-cmp-is="adaptiveFormTabs" id={id} className={`cmp-tabs ${appliedCssClassNames||''}`} data-placeholder-text="false" data-cmp-visible={visible} data-cmp-enabled={enabled}>
-      {label?.visible && <label className="cmp-tabs__label" htmlFor={`${id}-widget`}>{label?.value}</label>}
-      {props?.description && <button aria-label='Toggle Button' className="cmp-tabs__questionmark" onClick={handleButtonClick}></button>}
-      {shortDescription && props?.tooltip && <div title='Help Text' data-cmp-visible={shortDescription} className='cmp-tabs__shortdescription'>{props?.tooltip}</div>}
-      <div aria-live="polite">
-        {longDescription && props?.description && <div title='Help Text' data-cmp-visible={longDescription} className="cmp-tabs__longdescription">{props?.description}</div>}
-      </div>
+    <div
+      data-cmp-is="adaptiveFormTabs"
+      id={id}
+      className={`cmp-tabs ${appliedCssClassNames || ''}`}
+      data-placeholder-text="false"
+      data-cmp-visible={visible}
+      data-cmp-enabled={enabled}
+    >
+      <FieldWrapper
```

Comment:
This is not a field, code is not readable. Please rename FieldWrapper accordingly

Reply:
changed